extensions/wikia/InsightsV2/specials/PopularPages.class.php

```diff
@@ -73,12 +73,12 @@ public function recache( $limit = false, $ignoreErrors = true ) {
 	 * @return array
 	 */
 	public function reallyDoQuery( $limit = false, $offset = false ) {
-		global $wgCityId;
+		global $wgCityId, $wgContentNamespaces;
 
 		$topArticles = DataMartService::getTopArticlesByPageview(
 			$wgCityId,
 			null,
-			null,
+			$wgContentNamespaces,
```

Comment:
by user I meant user who can do it ;) and actually there is one wiki which does not have 0 namespace in wgContentNamespaces: http://mum.mpb-test.wikia.com/wiki/AfW8